### PR TITLE
Allow projects to load on SDK 10.0.15063 and update dependencies

### PR DESCRIPTION
**Pull request by ericwj**
First step of #19.

Allow projects to load on SDK 10.0.15063 and update dependencies.

Also remove OnXxxChanged which is obsoleted and to be replaced with RaiseXxxChanged.

Temporarily add static methods that create DelegateCommands from a task async execute method. To be discussed how this is to be resolved going forward.

### CODE REVIEW

File: AdventureWorks.Shopper/AdventureWorks.UILogic/DelegateCommandHack.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+using Prism.Commands;
+
+namespace AdventureWorks.UILogic {
+    /// <summary>Bring back FromAsyncHandler temporarily to limit the number and extend of changes.</summary>
+    public class DelegateCommandHack
```
Comment (brianlagunas) [inline AdventureWorks.Shopper/AdventureWorks.UILogic/DelegateCommandHack.cs]: This isn't needed.  All commands can be updated to just 

```
SaveCommand = new DelegateCommand(async () => await SaveAsync);
```

File: AdventureWorks.Shopper/AdventureWorks.UILogic/ViewModels/ItemDetailPageViewModel.cs [hunk 2]
```diff
@@ -110,7 +110,7 @@ public bool IsBottomAppBarOpened
                 // AppBar.IsOpen property doesn't notify when the property is set.
                 // See http://go.microsoft.com/fwlink/?LinkID=288840
                 _isBottomAppBarOpened = value;
-                OnPropertyChanged("IsBottomAppBarOpened");
+                RaisePropertyChanged("IsBottomAppBarOpened");
```
Comment (brianlagunas) [inline AdventureWorks.Shopper/AdventureWorks.UILogic/ViewModels/ItemDetailPageViewModel.cs]: we should use `RaisPropertyChanged(nameof(IsBottomAppBarOpened))`.  You can use find and replace for this. See this comment https://github.com/PrismLibrary/Prism/issues/1031#issuecomment-296704507

**Review by brianlagunas — CHANGES_REQUESTED** [review]
If you can make these changes I'll merge.

**Review by brianlagunas — APPROVED** [review]

**Comment by ericwj** [comment]
Mmm how do I still amend this? It doesn't build.

**Pull request merged**